crnlib/crn_assert.h

```diff
@@ -39,15 +39,7 @@ struct crnlib_assume_failure<true> {
 template <int x>
 struct crnlib_assume_try {};

```

Comment:
More CRNLIB_ASSUME bits that can be deleted above

Reply:
Done.